src/hooks/use-polkadot-api.hook.ts

```diff
@@ -64,7 +64,7 @@ export const usePolkadotApi = () => {
 
         if (api) {
           switch (availableTokens[i].id) {
-            case 'MYR':
+            case 'MYR': {
```

Comment:
bor @ical10  ini boleh mintol tambahin comment` // TODO: move to single file constant or enum` buat flag next task

Reply:
siap bor

Reply:
gue tambahin di PR selanjutnya aja ya mas @irmannmal , yang ini tolong di-merge dulu kalo udah OK

Reply:
ok ma bor